docusaurus/docs/tutorials/build-a-data-source-plugin.md

```diff
@@ -143,7 +141,7 @@ Next, we'll add the actual values to the data frame. Don't worry about the math
 
 1. Return the data frame:
 
-   ```ts
+   ```ts title="src/datasource.ts"
```

Comment:
below on line 152 we've got a random quote block, maybe we should make this a proper note/info?

Reply:
I've updated it to use an info admonition